app/assets/stylesheets/components/_home_info.scss

```diff
@@ -1,24 +1,28 @@
 .home-info-container{
-display: inline-flex;
-background-color: $light-blue;
-margin-left: 4%;
-margin-right: 4%;
-border:none;
-justify-content: center;
-align-items: center;
-border-radius: 10px;
+  display: inline-flex;
+  margin-left: 4%;
+  margin-right: 4%;
+  border:none;
+  justify-content: center;
+  align-items: center;
+  border-radius: 10px;
+  background-color: #c3e0e0;
+}
+.home-info-header {
+  margin-top: 5vh;
+  margin-bottom: 5vh;
+  color: $black;
 }
-
 .home-info-card {
   display: inline-flex;
   margin: 2%;
   border:none;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
```

Comment:
on mobile, main info banner is vertical, anything desktop and above becomes horizontal